.github/ISSUE_TEMPLATE/support-add-project-contributor.md

```diff
@@ -0,0 +1,21 @@
+---
+name: 'Support: Add me as Tractus-X project contributor'
+about: 'Add a user to our Eclipse Foundation project as an official contributor'
+title: 'New Tractus-X project contributor'
+labels: support
+assignees: ''
+---
+
+<!-- This issue template is used to request admission to the Eclipse Tractus-X project contributor group -->
+
+> __Before requesting the project contributor role, please make sure, that you:__
+>
+> - Make yourself familiar with the Eclipse project roles: https://eclipse-tractusx.github.io/docs/oss/contributor-committer
```

Comment:
Why not directly linking the Eclipse Project Handbook, just to ensure the right information is presented? A better link could be https://www.eclipse.org/projects/handbook/#contributing in regards of contributor requirements.

Reply:
The docs I linked do provide a condensed version of the handbook and is also referencing the handbook. So we would have easy to digest info + the full version of the handbook

Reply:
I wouldn't mind though, changing the URL. Any opinions from other reviewers?

Reply:
I would also like to share the same info as it already displayed on https://eclipse-tractusx.github.io/docs/oss/getting-started
so my vote for the Handbook in general.